cluster_tools/cluster_tools/schedulers/slurm.py

```diff
@@ -483,4 +525,4 @@ def run(self):
                     return
 
             job_id = SlurmExecutor.submit_text(script, self.cfut_dir)
-            future.set_result(job_id)
+            future.set_result(str(job_id))
```

Comment:
How about changing the signature of `submit_text` to return the job id as a string? String based job IDs generalize better over different cluster environments, anyway.